src/javascript/app_2/App/Components/Layout/Header/account_info.jsx

```diff
@@ -1,12 +1,32 @@
-import PropTypes    from 'prop-types';
-import React        from 'react';
-import { localize } from '../../../../../_common/localize';
+import classNames        from 'classnames';
+import { CSSTransition } from 'react-transition-group';
+import PropTypes         from 'prop-types';
+import React             from 'react';
+import AccountSwitcher   from '../../Elements/account_switcher.jsx';
+import { IconArrow }     from '../../../../Assets/Common';
+import { localize }      from '../../../../../_common/localize';
 
-const AccountInfo = ({ balance, currency, loginid }) => (
+const AccountInfo = ({ balance, currency, loginid, is_dialog_on, toggleDialog }) => (
     <div className='acc-balance'>
-        <div className='acc-info'>
-            <p className='acc-balance-currency'>{`${(currency || '').toUpperCase()} ${localize('Account')}`}</p>
-            <p className='acc-balance-accountid'>{loginid}</p>
+        <div className='acc-switcher-container'>
+            <div className={classNames('acc-info', { 'show': is_dialog_on })} onClick={toggleDialog}>
+                <p className='acc-balance-currency'>{`${(currency || '').toUpperCase()} ${localize('Account')}`}</p>
```

Comment:
1. To achieve a proper translation, better to not take apart a phrase.
So it can be like `localize('[_1] Account', [...])`

2. According to the description on the card, we'll display the account type here instead of currency. (the class name seems inappropriate too)

Reply:
@binary-hamedanchi , regarding 1) I assume something like this would be fine? `{localize('[_1] Account', [(currency || '').toUpperCase()])}`